packages/entities/entities-routes/src/components/RouteForm.vue

```diff
@@ -247,30 +238,23 @@ const emit = defineEmits<{
   (e: 'notify', options: { message: string, type: string }): void
 }>()
 
-const currentConfigHash = ref<string>(
+const configType = ref<'basic' | 'advanced'>('basic')
+const configFlavor = ref<RouteFlavor>(
```

Comment:
```suggestion
const configFlavor = ref<RouteFlavor>(
   props.routeFlavors.expressions
      ? RouteFlavor.EXPRESSIONS
      : RouteFlavor.TRADITIONAL
)
```

Reply:
Same as above.

Reply:
I simplified this block in https://github.com/Kong/public-ui-components/pull/2111/commits/84c943bf0c308077cfb66eb3e88c21d3ee3ddadb.